bin/doctrine

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env php
```

Comment:
this file is needed to validate the db schema

https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/tools.html#doctrine-console